Source/diablo.cpp

```diff
@@ -407,7 +407,7 @@ void RightMouseDown(bool isShiftHeld)
 {
 	LastMouseButtonAction = MouseActionType::None;
 
-	if (gmenu_is_active() || sgnTimeoutCurs != CURSOR_NONE || PauseMode == 2 || MyPlayer->_pInvincible) {
+	if (gmenu_is_active() || sgnTimeoutCurs != CURSOR_NONE || PauseMode == 2 || MyPlayer->_pInvincible || MyPlayer != InspectPlayer) {

```

Comment:
Why check here `InspectPlayer`?
The check could be done in `UseInvItem` (similar to `CheckInvItem`).
This would also help with enabling this feature for gamepad & touch users.

Perhaps @StephenCWills could help with pointing out where other checks are needed for gamepad & touch support.

Reply:
I tried to move the check later down the path so it should also work for gamepad and touch controls.
But I couldn't test it. :-/